### PR TITLE
store timestamp arrays instead of indices

### PR DESCRIPTION
**Pull request by JaerongA**

### CODE REVIEW

File: aeon/dj_pipeline/analysis/visit_analysis.py [hunk 1]
```diff
@@ -366,7 +368,7 @@ def make(self, key):
                         **food_patch_key,
                         "visit_date": visit_date.date(),
                         "time_fraction_in_patch": in_patch.mean(),
-                        "in_patch": in_patch.values,
+                        "in_patch": np.array(in_patch.index[np.where(in_patch)]),
```
Comment (ttngu207) [inline aeon/dj_pipeline/analysis/visit_analysis.py:371]:
```suggestion
                        "in_patch": np.array(in_patch.index[in_patch]),
```
would this work instead?

File: aeon/dj_pipeline/analysis/visit_analysis.py [hunk 2]
```diff
@@ -378,9 +380,9 @@ def make(self, key):
                     "visit_date": visit_date.date(),
                     "day_duration": day_duration,
                     "time_fraction_in_corridor": in_corridor.mean(),
-                    "in_corridor": in_corridor.values,
+                    "in_corridor": np.array(in_corridor.index[np.where(in_corridor)]),
```
Comment (ttngu207) [inline aeon/dj_pipeline/analysis/visit_analysis.py:383]:
```suggestion
                    "in_corridor": in_corridor.index.values[in_corridor],
```

File: aeon/dj_pipeline/analysis/visit_analysis.py [hunk 3]
```diff
@@ -378,9 +380,9 @@ def make(self, key):
...
                     "time_fraction_in_arena": in_arena.mean(),
-                    "in_arena": in_arena.values,
+                    "in_arena": np.array(in_arena.index[np.where(in_arena)]),
```
Comment (ttngu207) [inline aeon/dj_pipeline/analysis/visit_analysis.py:385]:
```suggestion
                    "in_arena": in_arena.index.values[in_arena],
```

Comment (JaerongA) [reply aeon/dj_pipeline/analysis/visit_analysis.py:385]: Yes, that seems better. Will accept those changes. Thanks.

**Review by ttngu207 — APPROVED** [review]

**Pull request merged**